spec/lib/js_fetcher_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+describe JsFetcher do
+  # Below live site will pinged on each test run, should be handled with VCR cassette config.
+  let(:url) { 'https://search.gov/javascript-test.html' }
+
+  it 'fetches page with javascript response' do
+    js_response = described_class.fetch(url)
+    expect(js_response).not_to be_nil
```

Comment:
Now that we've got a stronger expectation set on line 10, the not nil expectation on ln 9 seems unnecessary.  Go ahead an remove it, so that there's only one expectation in this block.